### PR TITLE
use custom networks assets

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)

- adding assets by address with auto fill symbol, decimals and asset name
- custom network assets with appear in assets tab

## Screen recordings / screenshots

https://www.loom.com/share/6032cdd75e674883b75640772ce7b92e

### CODE REVIEW

File: src/core/utils/assets.ts [hunk 1]
```diff
@@ -363,3 +363,59 @@ export const createAssetQuery = (
...
+  // parsedAsset,
+  assetAddress,
+  currentAddress,
+  // currency,
+  provider,
+}: {
+  // parsedAsset: ParsedUserAsset;
+  assetAddress: Address;
+  currentAddress: Address;
+  // currency: SupportedCurrencyKey;
```
Comment (brunobar79) [inline src/core/utils/assets.ts:402]: why are all these commented?

**Review by brunobar79 — APPROVED** [review]
🚀 🚀 🚀

File: src/core/utils/assets.ts (hunk 1 above)
Comment (estebanmino) [reply src/core/utils/assets.ts:402]: will remove, the method started with that and then i removed it

File: src/entries/popup/pages/settings/customChain/customChain.tsx [hunk 2]
```diff
@@ -68,6 +73,25 @@ export function CustomRPC() {
...
+        setAsset({
+          address: asset.address,
+          symbol: metadata.symbol || INITIAL_ASSET.symbol,
+          decimals: metadata.decimals || INITIAL_ASSET.decimals,
+          name: metadata.name || INITIAL_ASSET.name,
+        });
+      }
+    };
+    fetchMetadata();
+  }, [asset.address, chainId]);
```
Comment (greg-schrammel) [inline src/entries/popup/pages/settings/customChain/customChain.tsx:93]: why not a `useQuery`?

**Review by greg-schrammel — APPROVED** [review]

File: src/entries/popup/pages/settings/customChain/customChain.tsx (hunk 2 above)
Comment (estebanmino) [reply src/entries/popup/pages/settings/customChain/customChain.tsx:93]: oh yes, will use useQuery

Comment (estebanmino) [reply src/entries/popup/pages/settings/customChain/customChain.tsx]: ok done https://www.loom.com/share/9def5ef7d77949c68736f4fb0b748775

**Pull request merged**